server/src/repository/handover.repository.ts

```diff
@@ -0,0 +1,38 @@
+import { UpdateResult } from "typeorm";
+import { Handover } from "../entity/handover";
+
+const field: (keyof Handover)[] = ["id", "userId", "sysmoney", "realcash", "createdAt"];
+
+export class HandoverRepository {
+  async getAll(): Promise<Handover[]> {
+    return await Handover.find({
+      where: {
+        status: 0,
```

Comment:
handover table does not contain status property.

Reply:
I have added to the form